pkg/cmd/api/api_test.go

```diff
@@ -265,7 +265,7 @@ func Test_apiRun(t *testing.T) {
 			stderr: "gh: THIS IS FINE (HTTP 400)\n",
 		},
 		{
-			name: "GraphQL error",
+			name: "REST object error",
```

Comment:
This is a test for a GraphQL error response. Note the request path is `graphql`. I don't think this test case needed renaming

Reply:
Ok, I will restore the original name.